src/main/java/org/apache/sling/servlethelpers/MockSlingHttpServletResponse.java

```diff
@@ -262,6 +262,11 @@ public void setLocale(Locale loc) {
         this.locale = loc;
     }
 
+    public String getStatusMessage() {
+        return statusMessage;
+    }
+    
+    @Deprecated
```

Comment:
Please always give a short reasoning in javadoc what should be used instead of the deprecated method.
Also it would be good to call getStatusMessage(), so in case the latter method body is modified it affects both methods!

Reply:
+1 on both accounts. Note that the missing @deprecated javadoc tag was also noted in the SonarCloud run at https://github.com/apache/sling-org-apache-sling-servlet-helpers/pull/5/checks?check_run_id=131526044

Reply:
OK, I have updated the code as requested.  Thanks for the feedback.  Please let me know if you have any issues.